wsmaster/che-core-api-devfile/src/main/resources/build_docs.sh

```diff
@@ -62,23 +62,20 @@ build_native() {
 
 deploy() {
    cd ${TMP_DIR}
-   BRANCH_ARG=""
-   if [[ ! -z "${DEPLOY_BRANCH}" ]]; then
```

Comment:
I think it makes sense to leave this logic but rename `DEPLOY_BRANCH` to `BASE_BRANCH`